Assets/Scripts/Actors/Buddies/CinematicBuddyAnimationTrigger.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[RequireComponent( typeof( Animator ) )]
+public class CinematicBuddyAnimationTrigger : MonoBehaviour
+{
+	public string[] _triggerNames = null;
+
+	static int triggerIndex = 0;
+
+	void Start () 
+	{
+		string triggerName = _triggerNames[ triggerIndex ];
+
+		Animator animator = GetComponent<Animator>();
+		animator.Play( triggerName );
+
+		// Make it kind of random while also encouraging it to move through the list and not hit the same thing twice
+		triggerIndex += Random.Range( 1, 3 );
```

Comment:
If you want to do this in random order it'd probably be better to just shuffle the list and play it in random order.